### PR TITLE
Some additions to the new hires checklist

### PR DESCRIPTION
**Pull request by diegopetrucci**

### CODE REVIEW

File: TechnicalDocuments/NewHiresCheckList.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Here's how to get the iOS project up and running.
      git clone git@github.com:Babylonpartners/babylon-ios.git
      ```
 
+1. Get our latest Xcode templates by running `./Templates/install_xcode_templates.sh`
```
Comment (ilyapuchka) [inline TechnicalDocuments/NewHiresCheckList.md]: we also have some git hooks in `githooks` folder

**Comment by AliSoftware** [comment]
We should also list the services for which to ask permission for

| Service | Who to ask |
|---|---|
| 1Password | ? |
| JIRA | IT Support |
| Zeplin | ? |
| GitHub | You already got it |
| Confluence Spaces (iOS, your Tribe, ...) | Your PM / Tribe Lead? |
| Google Team Drives | PM? |
| VPN (esp if working remote) | IT Support |
| CircleCI | Use your GitHub login |

...

@inamiy since you joined recently like me, do you remember any other services for which you had to request access?

File: TechnicalDocuments/NewHiresCheckList.md (hunk 1 above)
Comment (ilyapuchka) [reply TechnicalDocuments/NewHiresCheckList.md]: I've started to work on a hook to install updates to the templates automatically btw

**Comment by ilyapuchka** [comment]
@AliSoftware this table is a must have! We should add Heroku there

**Comment by inamiy** [comment]
@AliSoftware +Hockey too!

**Comment by AliSoftware** [comment]
Maybe also links to some processes we use (SSDLC and such, linking [here](https://engineering.ops.babylontech.co.uk/) too?

**Comment by MichalTKwiecien** [comment]
Since I've done it yesterday, this is how I've obtained the access to some of those things:
* 1Password: IT Support
* Zeplin (any other person with the access)
* Confluence Spaces (Atlassian account): IT Support

**Comment by TheAdamBorek** [comment]
@AliSoftware I think we can add lists of apps we need an access to however I would create a separate file for this

**Comment by scass91** [comment]
From a hardware perspective - would it be worth referencing how to obtain headphones for video meetings? It was mentioned in the mini-induction that a budget is provided for those who need it, but no further information was given.

**Comment by scass91** [comment]
Slight tangent - alongside asking to be invited to any private slack channels, also suggest new hires ask for invites to any upcoming/recurring meetings?

**Comment by mluisbrown** [comment]
>  would it be worth referencing how to obtain headphones for video meetings?

@scass91 I think you can buy your own headphones up to 80 GBP and then get the cost refunded. I'm not sure exactly what the procedure for that is, just a normal expense via Concur or something else?

**Comment by Ben-Henshall** [comment]
@scass91 @mluisbrown 
>Please be respectful of your peers and try to keep the volume to a minimum. Be particularly aware of your teammates who sit near the stand-up area and the kitchen who suffer the most. If the noise levels are an issue for you, and you are a permanent employee we would like to contribute £80 towards a purchase of noise canceling headphones only. Send the receipt to <accounts email address> after buying them and we will pay you £80 in your next pay.

File: TechnicalDocuments/NewHiresCheckList.md (hunk 1 above)
Comment (Ben-Henshall) [reply TechnicalDocuments/NewHiresCheckList.md]: We do?

**Comment by AliSoftware** [comment]
@diegopetrucci Pushed new commit.
 - As per @TheAdamBorek 's suggestion, I've put the table in a separate file (and moved the instructions to Charles there too). 
 - As per @scass91 's suggestion I've added a bullet point to ask to be invited to recurring meetings.

Also @scass91 @MichalTKwiecien don't hesitate to confirm that my table in `ToolsAndServices.md` is correct especially about who to ask (I've only joined 4 weeks ago but already forgot who I asked back then ^^)

**Comment by AliSoftware** [comment]
Should we change `bundle exec pod install` to `bundle exec pod install --repo-update` in the instructions above @mluisbrown ?

File: TechnicalDocuments/ToolsAndServices.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Tools and Services
+
+These are tools and services used by the BabylonHealth's Consumer Apps tribe.
```
Comment (RuiAAPeres) [inline TechnicalDocuments/ToolsAndServices.md]: There are others, working in different tribes, that would still use these tools.

File: TechnicalDocuments/ToolsAndServices.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## Access to Services
+
+| Service | What for | How to access |
+|---|---|---|
+| Slack | Chat with whole org. See channels of interest [here](NewHiresCheckList.md#slack). | You should have given access during your mini induction. Log in with your BabylonHealth email and Okta |
+| 1Password | Access shared passwords used by various services in the team | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) |
+| JIRA | Tickets of tasks and issues on your sprint | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) |
+| Zeplin | Screen Designs | Ask `@Carlos` or `@adrian` on Slack |
+| GitHub | Repos for the app, the bots, the playbook... | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/248) to get added to the org. |
+| [Lokalize](https://lokalise.co/projects) | Manage translations used by the app | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) or your Team lead |
```
Comment (daniloaliberti) [inline TechnicalDocuments/ToolsAndServices.md]:
```suggestion
| [Lokalise](https://lokalise.co/projects) | Manage translations used by the app | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) or your Team lead |
```

**Comment by inamiy** [comment]
- https://realtimeboard.com : (I don't know who to ask)
- Debugging tips
    - Shake to auto-fill forms
    - Cmd-d for passing identity screen

**Comment by MichalTKwiecien** [comment]
👆 I've removed small mistake with duplicated channel name

File: TechnicalDocuments/NewHiresCheckList.md [hunk 4]
```diff
@@ -84,15 +97,22 @@ Here's how to get the iOS project up and running.
...
-     
+
 1. Configure the Xcode **Text Editing -> Indentation** preferences as follows:
      - Prefer indent using: Spaces
      - Tab width: 4 spaces
      - Indent width: 4 spaces
      - Tab key: Indents in leading whitespace
-     
+
 1. Compile the project 🎉
```
[the remark this answers is not in the document]
Comment (mluisbrown) [reply TechnicalDocuments/NewHiresCheckList.md]: You don't need the fastlane match config for developing on the simulator and starting to get work done.

File: TechnicalDocuments/ToolsAndServices.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# Tools and Services
+
+## Optional tools
+
+### Charles
+
+* Install Charles Proxy
```
Comment (mluisbrown) [inline TechnicalDocuments/ToolsAndServices.md]: Should mention that we have a site license for this (it's in 1Password vault).

**Pull request merged**